pkg/sql/pg_extension.go

```diff
@@ -32,6 +36,64 @@ var pgExtension = virtualSchema{
 	validWithNoDatabaseContext: false,
 }
 
+func postgisColumnsTablePopulator(
```

Comment:
The paradigm we have is to define the populate function normally, and call out to a function that takes in `addRow` instead -- I think its a bit cleaner that way.

Reply:
hmm, the problem then is i'd be duplicating the same code twice. seems cleaner to define this just once.

Reply:
Valid